nipoppy_cli/nipoppy/cli/run.py

```diff
@@ -44,24 +38,32 @@ def cli(argv: Sequence[str] = None) -> None:
         dpath_root = args.dataset_root
 
         if command == COMMAND_INIT:
+            from nipoppy.workflows.dataset_init import InitWorkflow
+
```

Comment:
maybe worth mentioning at the top of the files (or next to each import) why those import are there : it goes against the convention of putting them all at the top so it may confuse future coders